lua/neogit/lib/collection.lua

```diff
@@ -58,8 +58,8 @@ function M.reduce(tbl, func, ...)
 end
 
 function M.find(tbl, func)
```

Comment:
I don't think we need this module, we can just use `plenary/iterators.lua`. It is more general because it works on iterators and not tables which is probably faster too.

Reply:
This probably belongs in a seperate PR, because this was introduced before this PR.